iml-services/iml-device/src/main.rs

```diff
@@ -13,8 +14,12 @@ use iml_device::{
     ImlDeviceError,
 };
 use iml_orm::{
-    models::{ChromaCoreDevice, NewChromaCoreDevice},
-    schema::chroma_core_device::{devices, fqdn, table},
+    models::{ChromaCoreDevice, ChromaCoreLustreclientmount, NewChromaCoreDevice},
+    schema::{
+        chroma_core_device as dvc, chroma_core_lustreclientmount as clmnt,
+        chroma_core_managedfilesystem as fs, chroma_core_managedhost as host,
+        django_content_type as djct,
+    },
     tokio_diesel::*,
```

Comment:
```suggestion
    tokio_diesel::{AsyncRunQueryDsl as _, OptionalExtension as _},
```

Needed for `optional()` change below.